backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -185,23 +216,33 @@ export const updateCamperDtoValidator = async (
   if (req.body.hasLaptop && !validatePrimitive(req.body.hasLaptop, "boolean")) {
     return res.status(400).send(getApiValidationError("hasLaptop", "boolean"));
   }
-  if (
-    req.body.earlyDropoff &&
-    (!validatePrimitive(req.body.earlyDropoff, "string") ||
-      !validateTime(req.body.earlyDropoff))
-  ) {
-    return res
-      .status(400)
-      .send(getApiValidationError("earlyDropoff", "24 hr time string"));
+  if (!Array.isArray(req.body.earlyDropoff)) {
+    return res.status(400).send("early dropoff is not an array");
   }
-  if (
-    req.body.latePickup &&
-    (!validatePrimitive(req.body.latePickup, "string") ||
-      !validateTime(req.body.latePickup))
-  ) {
-    return res
-      .status(400)
-      .send(getApiValidationError("latePickup", "24 hr time string"));
+  // eslint-disable-next-line no-restricted-syntax
+  for (const dropoffDate of req.body.earlyDropoff) {
+    if (
+      !validatePrimitive(dropoffDate, "Date string") ||
+      !validateTime(dropoffDate)
```

Comment:
Should we be using validateDate here? (i.e. we aren't storing times like "12:30")